src/rules/rulepanel.ts

```diff
@@ -6,13 +6,16 @@
  * ------------------------------------------------------------------------------------------ */
 'use strict';
 
+import { capitalize } from 'lodash';
 import * as VSCode from 'vscode';
 import { ShowRuleDescriptionParams } from '../lsp/protocol';
 import * as util from '../util/util';
 import { clean, escapeHtml, ResourceResolver } from '../util/webview';
 import { decorateContextualHtmlContentWithDiff } from './code-diff';
 import { highlightAllCodeSnippetsInDesc } from './syntax-highlight';
 
+const CLEAN_CODE_CONCEPTS_URL = "https://sonar-documentations-preview.netlify.app/sonarlint/vs-code/concepts/clean-code/";
```

Comment:
Could we please add a TODO and/or create a ticket to update the link so it points to the released documentation once we have it?